modules/data.atmosphere/R/download.CRUNCEP_Global.R

```diff
@@ -113,16 +116,16 @@ download.CRUNCEP <- function(outfolder, start_date, end_date, site_id, lat.in, l
       }
 
 
-      dat.list[[j]] <- ncdf4::ncvar_get(dap,
+      dat.list[[j]] <- retry.func(ncdf4::ncvar_get(dap,
```

Comment:
Will we need to wrap a retry around *every* ndcf4 function that could potentially be used on a remote file? That seems like it will get messy.

Reply:
I don't think it will get too messy, there's really not THAT many remote file connections in the system and while it would be nice to be preventative, only a subset of those have been a source of time out issues.  I think this PR is a really useful addition for those cases, and there are places we know this will be an issue that we can get ahead of now (e.g. @annethomas fixes to the MODIS function)